coalib/parsing/DefaultArgParser.py

```diff
@@ -251,6 +251,16 @@ def default_arg_parser(formatter_class=None):
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    outputs_group.add_argument(
+        '--profile', nargs='?', const=True,
+        help='Enable bear profiling with cProfile. To specify where to dump the'
+        ' profiled files, supply the directory path. If specified directory'
+        ' does not exist it will be created. If the specified path points to an'
+        ' already existing file a ``SystemExit`` is raised. All bears(even'
```

Comment:
The user reading a help message on their console is never going to see `SystemExit` .  That is internal, and does not belong in the help message.  I already explained this.  `--help` needs to be written from the perspective of a CLI users -- what do *they* experience.  they get logs, and an exit code.